src/main/resources/static/javascript/mainApp/grouping.controller.js

```diff
@@ -1647,31 +1647,30 @@
          *                                   If "page", it toggles checkboxes on the current page; if "other", it updates event listeners.
          */
         $scope.toggleCheckAllSelection = (group, checkboxClicked) => {
-            let item;
             if (group === "Exclude") {
-                item = {
+                $scope.item = {
                     allItems: $scope.groupingExclude,
                     pageItems: $scope.pagedItemsExclude,
                     pageNumber: $scope.currentPageExclude
                 };
             } else if (group === "Include") {
-                item = {
+                $scope.item = {
                     allItems: $scope.groupingInclude,
                     pageItems: $scope.pagedItemsInclude,
                     pageNumber: $scope.currentPageInclude
                 };
             }
             if (checkboxClicked === "page") {
                 $scope.pageSelected = !$scope.pageSelected;
-                for (let member of item.pageItems[Number(item.pageNumber)]) {
+                for (let member of $scope.item.pageItems[Number($scope.item.pageNumber)]) {
                     $scope.membersInCheckboxList[member.uhUuid] = $scope.pageSelected;
                 }
-                let check = $scope.checkMainSelectAllCheckbox(item);
+                let check = $scope.checkMainSelectAllCheckbox($scope.item);
```

Comment:
These two lines can be const too.

Reply:
I changed line 1668 and got rid of that Number type converter. Is there another place I'm missing? You mentioned that there are two lines to change `let` to `const`.

Reply:
Oh, now that I saw your other comment, you're referring to inside the `for` loop. I didn't know you could use `const` in this syntax for iteration, thank you.